src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 0;
```

Comment:
Consider declaring the 'rentPerDay' variable to store the daily rent cost (40) and use it in calculations for better maintainability